packages/mdc-side-sheet/README.md

```diff
@@ -1,4 +1,318 @@
+    <div><label><input type="checkbox"> Work</label></div>
+  </div>
+</aside>
+```
+
+#### Menu Icon
+
+We recommend using [Material Icons](https://material.io/tools/icons/) from Google Fonts:
+
+```html
+<head>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+```
+
+However, you can also use SVG, [Font Awesome](https://fontawesome.com/), or any other icon library you wish.
+
+### Styles
+
+```scss
+@import "@material/side-sheet/mdc-side-sheet";
+```
+
+### JavaScript Instantiation
+
+```js
+import {MDCSideSheet} from "@material/side-sheet";
+const sideSheet = MDCSideSheet.attachTo(document.querySelector('.mdc-side-sheet'));
+```
+
+## Variants
+
+### Dismissible Side Sheet
+
+Dismissible side sheets are by default hidden off screen, and can slide into view. Dismissible side sheets should be used when navigation is not common, and the main app content is prioritized.
+
+```html
+<body>
+  <aside class="mdc-side-sheet mdc-side-sheet--dismissible">
+    <div class="mdc-side-sheet__content">
+      <div><label><input type="checkbox"> Events</label></div>
+      <div><label><input type="checkbox" checked> Personal</label></div>
+      <div><label><input type="checkbox" checked> Projects</label></div>
+      <div><label><input type="checkbox"> Reminders</label></div>
+      <div><label><input type="checkbox"> Work</label></div>
+    </div>
+  </aside>
+
+  <div class="mdc-side-sheet-app-content">
+    App Content
+  </div>
+</body>
+```
+
+> Apply the `mdc-side-sheet-app-content` class to the sibling element after the side-sheet for the open/close animations to work.\
+
+#### Usage with Top App Bar
+
+In cases where the side-sheet occupies the full viewport height, some styles must be applied to get the dismissible side-sheet and the content below the top app bar to independently scroll and work in all browsers.
+
+In the following example, the `mdc-side-sheet__content` and `main-content` elements should scroll independently of each other. The `mdc-side-sheet--dismissible` and `mdc-side-sheet-app-content` elements should then sit side-by-side. The markup looks something like this:
+
+```html
+<body>
+  <aside class="mdc-side-sheet mdc-side-sheet--dismissible">
+    <div class="mdc-side-sheet__content">
+      <div class="mdc-side-sheet__content">
+        <div><label><input type="checkbox"> Events</label></div>
+        <div><label><input type="checkbox" checked> Personal</label></div>
+        <div><label><input type="checkbox" checked> Projects</label></div>
+        <div><label><input type="checkbox"> Reminders</label></div>
+        <div><label><input type="checkbox"> Work</label></div>
+      </div>
+    </div>
+  </aside>
+
+  <div class="mdc-side-sheet-app-content">
+    <header class="mdc-top-app-bar app-bar" id="app-bar">
+      <div class="mdc-top-app-bar__row">
+        <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
```

Comment:
This should be updated to use an icon at the end of the top app bar consistent with the side sheet design article. e.g.:

```html
    <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end" role="toolbar">
      <a href="#" class="material-icons mdc-top-app-bar__action-item" aria-label="Filters" alt="Filters">filter_list</a>
    </section>
```

Could we also update this in the screenshot test pages? (That could be in a separate task if you really want.)